src/content/peerconnection/pr-answer/js/main.js

```diff
@@ -24,7 +24,9 @@ hangUpButton.disabled = true;
 
 let pc1 = null;
 let pc2 = null;
-let localstream;
+let localStream;
+const remoteStream = new MediaStream()
```

Comment:
Missing ;

Reply:
Fixed. Lets now see if travis is happy :)